cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForGenerators.java

```diff
@@ -112,9 +112,13 @@ private boolean setRegulatingControlVoltage(String controlId,
 
         // add qPercent as an extension
         if (!Double.isNaN(qPercent)) {
-            gen.newExtension(CoordinatedReactiveControlAdder.class)
-                    .withQPercent(qPercent)
-                    .add();
+            if (qPercent < 0.0 || qPercent > 100) {
+                context.ignored(QPERCENT, () -> String.format("Unexpected value for qPercent of generator %s: %s", gen.getId(), String.valueOf(qPercent)));
```

Comment:
Not sure valueOf is required. I think String.format is able to convert the value to a string